// #ifndef FS_COMM_HANDLERS_H_INCLUDED
// #define FS_COMM_HANDLERS_H_INCLUDED

// #include "../../proto-nanopb/file-system-commands.pb.h"
// #include "pbBufferCommProcessor.h"


// void handleRmFileCommand(RmFileCommand * cmd);
// void handleResetFilesystemCommand(ResetFilesystemCommand * cmd);

// void handleFileSystemCommand(FileSystemCommand * fsCommand, PbBufferCommProcessor * comm);
// void handleReportFileDataCommand(ReportFileDataCommand * cmd, PbBufferCommProcessor * comm);
// void handleStoreUpcomingFileCommand(StoreUpcomingFileDataCommand * cmd, PbBufferCommProcessor * comm);

// void handleListDirContents(ListDirContentsCommand * cmd, PbBufferCommProcessor * comm);
// void handlePrintDirContents(PrintDirContentsCommand * cmd, PbBufferCommProcessor * comm);
// void handlePrintFileContents(PrintFileContentsCommand * cmd, PbBufferCommProcessor * comm);


// #endif
